daml-lf/archive/proto_breaking_changes.sh

```diff
@@ -11,9 +11,13 @@ readonly config_file="${arch_dir}/buf.yaml"
 readonly stable_dir="${arch_dir}/src/stable/protobuf"
 readonly main_dir="${arch_dir}/src/main/protobuf"
 
-#We check the directories contains exactly 2 proto files
-for dir in "${stable_dir}" "${main_dir}"; do
+# We check that the stable directories contain exactly 3 proto files
```

Comment:
the comment was already bad, do you know what are the 2/3 expected proto files? 1.14, 1.15 and now 2.0? something else?

Reply:
We need to rethink this whole buf thing anyway, it's half-broken (see PR #17347)